Install static assests for the labextension (#67)
jupyter-widgets/widget-cookiecutter: {{cookiecutter.github_project_name}}/README.md

# {{ cookiecutter.github_project_name }}

{{ cookiecutter.project_short_description }}

## Installation

### User installation:

```bash
pip install {{ cookiecutter.python_package_name }}
jupyter nbextension enable --py --sys-prefix {{ cookiecutter.python_package_name  }}
```

#### Install the JupyterLab extension

In order to install the JupyterLab extension, you will first need to ensure `nodejs` is available in your environment. You can install it from [the nodejs website](https://nodejs.org/en/download) or using `conda`:
```bash
conda install -c conda-forge nodejs
```

You will also need to install the Widgets Manager for JupyterLab with:

```bash
jupyter labextension install @jupyter-widgets/jupyterlab-manager
```

> If you already installed the Widgets Manager previously, you need to at least run the following command in order to install the {{ cookiecutter.python_package_name  }} labextension:

Reviewer: ```suggestion
If you have previously installed the Widgets Manager then you will need to run the following command in order to finish installing the {{ cookiecutter.python_package_name  }} labextension:
```
